client/src/components/App/Content/UsersManagement/UsersFilter/SearchBar.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { TextField, InputAdornment, IconButton } from '@material-ui/core';
+import { Close, Search } from '@material-ui/icons';
+
+import { stringAlphanum } from 'commons/AlphanumericTextField/AlphanumericTextField';
+
+import useStyles from './SearchBarStyles';
+
+const searchBarLabel = 'הכנס שם או שם משתמש...';
+const searchBarError = 'יש להכניס רק אותיות ומספרים';
+
+const SearchBar: React.FC<Props> = (props: Props) => {
+
+    const { onClick } = props;
+
+    const [searchQuery, setSearchQuery] = useState<string>('');
+    const [isQueryValid, setIsQueryValid] = useState<boolean>(true);
+
+    const classes = useStyles();
+
+    const handleChange = (value: string) => {
+        if (stringAlphanum.isValidSync(value)) {
+            setSearchQuery(value);
+            !isQueryValid && setIsQueryValid(true);
+        } else {
+            setIsQueryValid(false);
+        }
+    }
+
+    const onClearClick = () => {
+        setSearchQuery('');
+        !isQueryValid && setIsQueryValid(true);
+        onClick(''); 
+    }
+
+    return (
+        <TextField
```

Comment:
why not using AlphanumericTextField?

Reply:
I'm pretty sure it works only under form

Reply:
it use useFormContext which i don't have